moshi/src/main/java/com/squareup/moshi/JsonUtf8Writer.kt

```diff
@@ -265,7 +265,7 @@ internal class JsonUtf8Writer(
     var i = 1
     val size = stackSize
     while (i < size) {
-      sink.writeUtf8(indent)
+      sink.writeUtf8(indent!!)
```

Comment:
this shouldn't be necessary

Reply:
Smart cast to 'String' is impossible, because 'indent' is a mutable property that could have been changed by this time